src/normalize/entities.ts

```diff
@@ -483,3 +483,12 @@ export const ContactInputRequest = new Entity({
 	a: ['attributes', ContactInputAttributes],
 	m: 'memberOps'
 });
+
+const AddMsgAttributes = new Entity({
+	aid: 'attachmentId',
+	l: 'folderId'
```

Comment:
AddMsg API supports content in the request. So instead or uploading MIME first and then passing aid inside this API, we can directly pass content and folderId.

Reply:
Fixed, Thanks for pointing out this.